src/main/kotlin/com/kotlin/boardproject/domain/notification/domain/Notification.kt

```diff
@@ -1,6 +1,8 @@
-package com.kotlin.boardproject.model
+package com.kotlin.boardproject.domain.notification.domain
 
-import com.kotlin.boardproject.common.enums.NotificationType
+import com.kotlin.boardproject.global.enums.NotificationType
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end